applications/server/controllers/synchronization.py

```diff
@@ -102,12 +106,153 @@ def POST(*args, **vars):
 
     return locals()
 
+"""
+    This method is to receive value data.  It accepts two HTTP methods:
+        GET: returns all the current output entries (for the device id)
+        POST: accepts a JSON document with the value entries to ingest.
+            all current entries (for the device_id) will be removed and replaced
```

Comment:
Hey, for output, you should NOT remove all current entries.  We want to generate graphs of the output values, and this will not work otherwise. 

Question: how does the server know how much to synchronize, that is, how much to send up?